Spigot-API-Patches/0299-Add-corner-getter-to-WorldBorder.patch

```diff
@@ -0,0 +1,68 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TwoLeggedCat <80929284+TwoLeggedCat@users.noreply.github.com>
+Date: Wed, 12 May 2021 21:39:12 -0500
+Subject: [PATCH]  Add corner getter to WorldBorder
+
+
+diff --git a/src/main/java/org/bukkit/WorldBorder.java b/src/main/java/org/bukkit/WorldBorder.java
+index afb7b136b461202026290624836446cff9f9e45d..81cd8a824cf77d6431272cfa5095ae88028faf93 100644
+--- a/src/main/java/org/bukkit/WorldBorder.java
++++ b/src/main/java/org/bukkit/WorldBorder.java
+@@ -54,6 +54,15 @@ public interface WorldBorder {
+      */
+     public void setCenter(@NotNull Location location);
+ 
++    //Paper start
++    /**
++     * Returns the specified corner of the WorldBorder
++     * @param corner Which corner to return
++     * @return Location of corner where y is always the minimum height of the world, or null if Corner is null
++     */
++    public @NotNull Location getCornerLocation(@NotNull Corner corner);
++    //Paper end
++
+     /**
+      * Gets the current border damage buffer.
+      *
+@@ -121,7 +130,7 @@ public interface WorldBorder {
+     // Paper start
+     /**
+      * Checks if the location is within the boundaries of this border.
+-     * 
```

Comment:
Remove this change, just creates an unnecessary hunk.

Reply:
Not sure how I did that; there's an asterisk added in the line beneath it too.

Reply:
You removed a space at the end of the line it looks like.